ckanext/xloader/plugin.py

```diff
@@ -123,11 +116,23 @@ def notify(self, resource):
         )
         self._submit_to_xloader(resource_dict)
 
-    # IResourceController                
+    # IResourceController
+    if toolkit.check_ckan_version('2.10'):
```

Comment:
In this case the logic in the hooks is very simple so it it ok, but in general I think this is a better pattern to avoid duplication:

```python
class MyPlugin():

    # CKAN < 2.10
    def after_create(self, context, data_dict):
        return self.after_resource_create(context, data_dict)

    # CKAN >= 2.10
    def after_resource_create(self, context, data_dict):
        # Do whatever

```